pkg/apis/hive/v1/openstack/machinepools.go

```diff
@@ -5,7 +5,10 @@ package openstack
 type MachinePool struct {
 	// FlavorName defines the OpenStack Nova flavor.
 	// eg. m1.large
-	FlavorName string `json:"type"`
+	// The json key here differs from the installer which uses both "computeFlavor" and type "type" depending on which
+	// type you're looking at, and the resulting field on the MachineSet is "flavor". We are opting to stay consistent
+	// with the end result.
+	FlavorName string `json:"flavor"`
```

Comment:
What is the reasoning behind keeping the name of the field `FlavorName` instead of `Flavor` to match the json key?